### ISSUE TITLE
Is a .pypirc file required for zest.releaser?

### ISSUE THREAD

**Issue by mctwynne** [zestsoftware/zest.releaser#353]
I'd like to run zest.releaser from places where I don't necessarily have a `.pypirc`. Since twine supports the use of environment variables for the repository URL, username, and password, I assumed that setting those and running `fullrelease` would work.

However, I received this error:

```
ERROR: You must have a properly configured .pypirc file in your home dir to upload to a Python package index.
```

Am I missing something, or is this not supported by zest.releaser?

Thanks in advance!

**Comment by mctwynne** [comment]
@mauritsvanrees Thanks for getting back to me! It looks like the environment is already passed to twine, so that shouldn't be an issue. I will create a PR.

**Comment by mauritsvanrees** [comment]
I have released zest.releaser 6.21.0 with your fix from #354
